docs/Files/UI/Solution-Manager/Library/ServerOptions/Managing-SSO-Configurations.md

```diff
@@ -0,0 +1,207 @@
+| Y                   | Y               | Y                    | N                             | N                | No action since user cannot access the application        |
+| N                   | Y               | Y                    | N/A                           | Y                | No action since user does not belong to an existing group |
+| N                   | Y               | Y                    | N/A                           | N                | No action since user does not belong to an existing group |
+| N                   | N               | N/A                  | N/A                           | N/A              | No action                                                 |
+
+:::tip Group Mappings
+
+- Group name values should be unique.
+- Group names can be linked to many roles.
+- Roles can be linked to many group names.
+
+:::
+
+:::tip Example
+The following is an example of filling out the required fields:
+
+1. SSO Settings:
+   ![Server Options - SSO Settings Sample](../../../../../Resources/Images/SM/Library/ServerOptions/sso-settings-sample.png "SSO Settings Sample")
+1. Group Mappings:
+   ![Server Options - SSO Settings Sample](../../../../../Resources/Images/SM/Library/ServerOptions/sso-group-mappings-sample.png "SSO Settings Sample")
+
+:::
+
+---
+
+### Creating an applications that are OpenID Connect Compatible
+
+#### Okta Application
+
+This document will describe the steps needed to create a custom application in Okta to return custom optional claims in a access token that will be used by SMAOpConRestApi.
+
+1.  After you’ve logged in to Okta, click Admin in the upper right corner to go to the administration dashboard
+    - ![Okta-Application](../../../../../Resources/Images/SM/Library/ServerOptions/okta-initial.png "Okta - Landing Page")
+1.  Once on the administration dashboard, click “Applications” from the left navigation to go to Applications
+    - ![Okta-Application](../../../../../Resources/Images/SM/Library/ServerOptions/okta-application-start.png "Okta - Dashboard")
+1.  Click the "Create App Integration" button
+    - ![Okta-Application](../../../../../Resources/Images/SM/Library/ServerOptions/okta-app-integration.png "Okta - Applications")
+    1.  Inside the modal, select the following options: 1. Sign-in method: OIDC - OpenID Connect 1. Application type: Single-Page Application
+    - ![Okta-Application](../../../../../Resources/Images/SM/Library/ServerOptions/okta-app-integration.png "Okta - Integration Options")
+    1. Click on "Next"
+    1. Set the following options:
+       1. Grant type: Authorization
+       1. Sign-in redirects URIs:(sample) `https://<hostaname:443>/login/callback`
+          1. The **hostname** will be used in the SSO Configuration page in Solution Manager. Make sure to include the **port number** if applicable
```

Comment:
1. The **hostname** will be used [on] the SSO Configuration page in Solution Manager. Make sure to include the **port number** if applicable